### ISSUE TITLE
When opening windows remote desktop, VirtualSpace hotkeys take control of my remote desktop control

### ISSUE THREAD

**Issue by direrFlood** [newlooper/VirtualSpace#12]
When opening the windows remote desktop, VirtualSpace hotkeys take control of my remote desktop control instead of using remote desktop controls...
1) open windows remote desktop
2) remote desktop on any pc while making sure the setting: let remote desktop access all keyboard hotkeys when in use.
3) make sure VirtualSpace is running on the local pc (not the remote one).
4) use any VirtualSpace hotkey, for example, the windows + tab.
5) it triggers the VirtualSpace hotkey on the local pc instead of the task view on the remote desktop. Basically, I see the local desktop task view (VirtualSpace hotkey taking over) instead of the remote desktop task view.

In other words, 
**the expectation is:** to use remote desktop keyboard hotkeys/shortcuts.
**What actually happens:** VirtualSpace gets triggered by keyboard hotkeys/shortcuts instead of the local desktop.

**Temporary solution:** 
every time u need to use a remote desktop: exit/terminate VirtualSpace manually from the local pc. and then go to the remote desktop...

**Comment by newlooper** [comment]
<img src="https://user-images.githubusercontent.com/13162531/200447638-df503f82-a9af-4e32-90e0-bebc36f84efa.png" data-canonical-src="https://user-images.githubusercontent.com/13162531/200447638-df503f82-a9af-4e32-90e0-bebc36f84efa.png" width="400"  />
